bundles/org.openhab.binding.opensprinkler/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -123,6 +134,13 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="queuedZones">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
Why using `Number:Dimensionless`?
It is a simple number, no ?

Reply:
I have no reason. What is Number:Dimensionless for then and when would be be used over just Number?

Reply:
When your value has a unit but no defined dimension like for example percentage or decibel.